packages/react-components/src/components/Input/Input.tsx

```diff
@@ -1,34 +1,91 @@
 import * as React from 'react';
 import cx from 'clsx';
 
+import {
+  VisibilityOn as VisibilityOnIcon,
+  VisibilityOff as VisibilityOffIcon,
+} from '@livechat/design-system-icons/react/material';
+
 import styles from './Input.module.scss';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
 
-type InputSize = 'xsmall' | 'small' | 'medium' | 'large';
+type InputSize = 'compact' | 'medium' | 'large';
 
 export interface InputProps extends React.HTMLAttributes<HTMLInputElement> {
   size?: InputSize | undefined;
   error?: boolean | undefined;
   disabled?: boolean | undefined;
+  kind?: 'text' | 'password';
```

Comment:
To be honest, I'm not sure if it's a good approach. We already extend `React.HTMLAttributes<HTMLInputElement>` which has a prop called `type` And it should handle `password` type -> https://www.w3schools.com/html/html_form_input_types.asp

Reply:
☝️